packages/messaging/src/controllers/controller-interface.ts

```diff
@@ -14,33 +14,33 @@
  * limitations under the License.
  */
 
-import { ErrorFactory } from '@firebase/util';
-import { ERROR_CODES, ERROR_MAP } from '../models/errors';
-import { TokenDetailsModel } from '../models/token-details-model';
-import { VapidDetailsModel } from '../models/vapid-details-model';
-import { IIDModel } from '../models/iid-model';
-import { arrayBufferToBase64 } from '../helpers/array-buffer-to-base64';
 import { FirebaseApp } from '@firebase/app-types';
-import { TokenDetails } from '../interfaces/token-details';
 import {
   createSubscribe,
-  Observer,
+  ErrorFactory,
   NextFn,
+  Observer,
   PartialObserver
 } from '@firebase/util';
+import { arrayBufferToBase64 } from '../helpers/array-buffer-to-base64';
+import { TokenDetails } from '../interfaces/token-details';
+import { ERROR_CODES, ERROR_MAP } from '../models/errors';
+import { IIDModel } from '../models/iid-model';
+import { TokenDetailsModel } from '../models/token-details-model';
+import { VapidDetailsModel } from '../models/vapid-details-model';
 
 const SENDER_ID_OPTION_NAME = 'messagingSenderId';
 // Database cache should be invalidated once a week.
 export const TOKEN_EXPIRATION_MILLIS = 7 * 24 * 60 * 60 * 1000; // 7 days
 
 export abstract class ControllerInterface {
-  public app: FirebaseApp;
-  public INTERNAL: any;
+  app: FirebaseApp;
+  INTERNAL: any;
```

Comment:
These originally were public to ensure the Firebase SDK would work with the module (i.e. the name shouldn't be mangled).

Reply:
Public is the default, so they are still public. This is just removing the redundant "public" keywords.